mkosi

```diff
@@ -2852,7 +2916,13 @@ def make_tar(args: CommandLineArguments,
 
     with complete_step('Creating archive'):
         f: BinaryIO = cast(BinaryIO, tempfile.NamedTemporaryFile(dir=os.path.dirname(args.output), prefix=".mkosi-"))
-        run(["tar", "-C", root, "-c", "-J", "--xattrs", "--xattrs-include=*", "."],
+# some distributions defaults to bsdtar(libarchive) which uses POSIX argument list so let's keep a separate list
+        if shutil.which('bsdtar'):
+           _tar_cmd = ["bsdtar", "-cf", "-", "-J", "-C", root, "."]
```

Comment:
It would read nicer if the options for `_tar_cmd` would be in the same order as for GNU tar. `--xattrs` is the default when run as root, but maybe add it for completeness.

Reply:
libarchives's bsdtar follow POSIX standard so it has its own options order. libarchive is default tar implementation in OpenMandriva for 4 or 5 years.

Reply:
So, `["bsdtar", "-C", root, "-c", "-J", "--xattrs", "-f", "-", "."]` would be an illegal invocation of bsdtar?

Reply:
Fixed, thanks for the hint.